Source/LightboxViewCell.swift

```diff
@@ -0,0 +1,21 @@
+import UIKit
+
+public class LightboxViewCell: UICollectionViewCell {
+
+  public static let reuseIdentifier: String = "LightboxViewCell"
```

Comment:
I would put reuseIdentifier to the ViewController file. Because,
1. this reuse identifier is never used inside the cell, but rather is used to identify the cell by the ViewController, so in my opinion, it is better to configure the identifier somewhere near the place you do 
```Objective-C
    collectionView.registerClass(LightboxViewCell.self,
      forCellWithReuseIdentifier: LightboxViewCell.reuseIdentifier)
```
2. The example mentioned above is being used by Apple in all the templates.